packages/documentation-site/patternfly-docs/content/get-started/develop.md

```diff
@@ -136,3 +136,38 @@ PatternFly is supported on the 2 latest major versions of the following browsers
 - Firefox
 - Safari
 - Edge
+
+## Breaking changes
+
+A breaking change is any modification that is significant enough to require code updates in order to maintain proper functionality in a UI. To avoid major and unexpected disruptions, PatternFly typically reserves breaking changes for major version releases, but critical bug fixes can require breaking changes outside of the major release schedule. 
+
+Breaking changes can occur in both the code and the design of PatternFly features, including our beta offerings. 
```

Comment:
I think the exception about beta components/features allowing for breaking changes gets a little lost here. Since we have sections below for design, I wonder if we could combine this with the intro paragraph?

```
A breaking change is any modification that is significant enough to require code updates in order to maintain proper functionality in a UI, and can occur in both the code and design of PatternFly features. To avoid major and unexpected disruptions, PatternFly typically reserves breaking changes for major version releases, with the following exceptions:
- Critical bug fixes can require breaking changes outside of the major release schedule.
- Components and features labeled as "beta."
```